common/etl/buffers.go

```diff
@@ -0,0 +1,167 @@
+package etl
+
+import (
+	"bytes"
+	"sort"
+	"strconv"
+)
+
+const (
+	//SliceBuffer - just simple slice w
+	SortableSliceBuffer = iota
+	//SortableAppendBuffer - just simple slice w
+	SortableAppendBuffer
+
+	BufferOptimalSize = 256 * 1024 * 1024 /* 256 mb | var because we want to sometimes change it from tests */
+)
+
+type Buffer interface {
+	Put(k, v []byte)
+	Get(i int) sortableBufferEntry
+	Len() int
+	Reset()
+	GetEntries() []sortableBufferEntry
+	Sort()
+	CheckFlushSize() bool
+}
+
+type sortableBufferEntry struct {
+	key   []byte
+	value []byte
+}
+
+var (
+	_ = &sortableBuffer{}
```

Comment:
shouldn't it be `_ Buffer = &sortableBuffer{}` if you are checking the interface here?

Reply:
Yes, sure. Thank you!